src/KingSystem/ActorSystem/actBaseProcHandle.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <basis/seadTypes.h>
+#include "actBaseProcUnit.h"
+#include "KingSystem/Utils/Types.h"
+
+namespace ksys::act {
+
+class BaseProcUnit;
+
+class BaseProcHandle {
+public:
+    BaseProcHandle();
+    ~BaseProcHandle();
+
+    static BaseProcHandle sDummyBaseProcHandle;
```

Comment:
Could this be renamed to sDummyHandle? Since this is a static member of BaseProcHandle, the "BaseProc" in the name seems a bit redundant.